src/clients/FlexbaseClient.Company.ts

```diff
@@ -0,0 +1,38 @@
+import { FlexbaseClientBase } from './FlexbaseClient.Base';
+import IEmployee from '../models/Company/Employee';
+
+interface UserInfoPreview {
+  firstName?: string;
+  lastName?: string;
+  jobTitle?: string;
+  email?: string;
+  phone?: string;
+  id?: string;
+}
+
+export class FlexbaseClientCompany extends FlexbaseClientBase {
```

Comment:
Do we plan to have more company specific calls here or should this be rename to Employees?